resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,279 @@
+        altLabels: ['Subgenre'],
+        fallbackValue: game.genre,
+      }),
+    [allGameHubs, game.genre],
+  );
+
+  const languageRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Meta - Language -', 'Meta - Language Patch -'],
+        primaryLabel: 'Meta - Language',
+        altLabels: ['Meta - Language Patch'],
+        altLabelsLast: true,
+        markAltLabels: true,
+      }),
+    [allGameHubs],
+  );
+
+  const themeRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Theme -'],
+        primaryLabel: 'Theme',
+      }),
+    [allGameHubs],
+  );
+
+  const perspectiveRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Perspective -'],
+        primaryLabel: 'Perspective',
+      }),
+    [allGameHubs],
+  );
+
+  const featureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Feature -', 'Game Mechanic -'],
+        primaryLabel: 'Feature',
+        altLabels: ['Game Mechanic'],
+        altLabelsLast: false,
+      }),
+    [allGameHubs],
+  );
+
+  const creditRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Credits -'],
+        primaryLabel: 'Credits',
+      }),
+    [allGameHubs],
+  );
+
+  const technicalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Technical -'],
+        primaryLabel: 'Technical',
+      }),
+    [allGameHubs],
+  );
+
+  const miscRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Misc. -'],
+        primaryLabel: 'Misc.',
+      }),
+    [allGameHubs],
+  );
+
+  const protagonistRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Protagonist -'],
+        primaryLabel: 'Protagonist',
+      }),
+    [allGameHubs],
+  );
+
+  const settingRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Setting -'],
+        primaryLabel: 'Setting',
+      }),
+    [allGameHubs],
+  );
+
+  const regionalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Regional -'],
+        primaryLabel: 'Regional',
+      }),
+    [allGameHubs],
+  );
+
+  const raFeatureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: [
+          'Meta -',
+          'Rollout Sets -',
+          'AotW -',
+          'RANews -',
+          'RA Awards -',
+          'Custom Awards -',
+          'Console Wars',
+          'DevJam',
+          'Challenge League',
+        ],
```

Comment:
Not sure how easy it would be to reclassify `[Misc - Junior Developer Graduate Sets]`
![image](https://github.com/user-attachments/assets/a0e57f18-225e-4fce-9c7e-19a23a7db60b)
Or maybe the hub could be renamed to something that would get matched?

Reply:
This one is tricky, but I think this should probably be renamed from a Misc. hub to something else.

The first thing that comes to mind is "Junior Developer - Graduate Sets".

Reply:
I'm fine with that. Whatever the title ends up being, it feels like it should get grouped into the Events section with DevQuest sets.

Reply:
Or maybe we should have a separate Collections sub-category for Site Events?

Reply:
I don't like the idea of a "Collections" sub-category - this seems like it may be fairly nebulous.

On prod, I've moved the JrDev grad sets hub to Dev Events: https://retroachievements.org/manage/hubs/7975
Additionally, it now has "[Central - Developer Events]" as its only related hub.

With these changes and the code in latest, Junior Developer Graduate Sets now shows up in the "Events" row.